src/sentry/api/endpoints/organization_releases.py

```diff
@@ -502,14 +502,14 @@ def post(self, request: Request, organization) -> Response:
                     release.status = new_status
                     release.save()
 
-                new_projects = []
+                new_releaseprojects = []
                 for project in projects:
-                    created = release.add_project(project)
-                    if created:
-                        new_projects.append(project)
+                    obj, releaseproject_created = release.add_project(project)
```

Comment:
nit: if we're not using the `obj` we can replace it with `_`

Reply:
Yea i'll do that